examples/test/GeoPushpinChart_test.js

```diff
@@ -8,12 +8,12 @@ fixture("Geo Pushpin Chart Demo") // eslint-disable-line no-undef
     .page(config.url)
     .beforeEach(loginUsingLoginForm(`${config.url}/geo-pushpin-chart`));
 
-test("should render all charts", async t => {
-    const geoPushpinChartWithClustering = Selector(".s-geo-pushpin-chart-clustering");
-    const geoPushpinChartWithColorLegend = Selector(".s-geo-pushpin-chart-color");
-    const geoPushpinChartWithCategoryLegend = Selector(".s-geo-pushpin-chart-category");
-    const geoPushpinChartWithConfiguration = Selector(".s-geo-pushpin-chart-configuration");
+const geoPushpinChartWithClustering = Selector(".s-geo-pushpin-chart-clustering");
+const geoPushpinChartWithColorLegend = Selector(".s-geo-pushpin-chart-color");
+const geoPushpinChartWithCategoryLegend = Selector(".s-geo-pushpin-chart-category");
+const geoPushpinChartWithConfiguration = Selector(".s-geo-pushpin-chart-configuration");
```

Comment:
It could be
```
const CHART_WITH_CLUSTERING = Selector(".s-geo-pushpin-chart-clustering");
const CHART_WITH_COLOR_LEGEND = Selector(".s-geo-pushpin-chart-color");
const CHART_WITH_CETEGORY_LEGEND = Selector(".s-geo-pushpin-chart-category");
const CHART_WITH_CONFIGURATION = Selector(".s-geo-pushpin-chart-configuration");
```

Reply:
fixed